src/routes/physicalCustomer.route.js

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express'
+
+import {
+  createPhysicalCustomer,
+  deletePhysicalCustomer,
+  getPhysicalCustomers,
+  updatePhysicalCustomer
+} from '../controllers/physicalCustomer.controller.js'
+
+const router = Router()
+
+router.get('/:id?', getPhysicalCustomers)
```

Comment:
Should we have different routes for each (get unique, get all) & specifics imports to have the same agreement :

get an UNIQUE element & another for get ALL elements

Reply:
irrelevant, carry over to next sprint🤙🏻

Reply:
reported to human resources